src/edu/wright/cs/raiderplanner/controller/ChatController.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2018
+ *
+ *
+ *
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package edu.wright.cs.raiderplanner.controller;
+
+import javafx.event.ActionEvent;
+import javafx.geometry.Insets;
+import javafx.scene.control.Button;
+import javafx.scene.control.TextArea;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.paint.Color;
+
+/**
+ * This is a class to handle the code for the chat feature.
+ * @author MichaelPantoja
+ */
+public class ChatController {
+
+	//chat variables
+	private static final GridPane userMessagePane = new GridPane();
+	private static final HBox spacingBox = new HBox();
+	private static TextField tfMessageToSend = new TextField();
+	private static TextArea msgArea = new TextArea();
+	private static final Button sendButton = new Button("Send");
+
+	/**
+	 * Default Constructor.
+	 */
+	public ChatController() {
+	}
+
+	/**
+	 * This will set the message area to uneditable and set the size for all the buttons This method
```

Comment:
Please correct the sentence structure and punctuation in this comment.

Reply:
@rsanchez-wsu This was not a function or comment I wrote. This was copied from MenuController. I'll still fix the problem.